pkg/kubectl/cmd/taint.go

```diff
@@ -366,23 +382,20 @@ func validateNoTaintOverwrites(obj runtime.Object, taints []v1.Taint) error {
 }
 
 // updateTaints updates taints of obj
```

Comment:
I am just reorganizing in this PR. I will create one more with some other modifications.